src/index.html

```diff
@@ -45,6 +47,65 @@
       </div>
     </div>
 
-  </div>
+
+    <div class = "titleframe">
+      <button class="button button__input">Title:</button>
+      <input class ="title__input">
+    </div>
+
+    <div class ="bodyframe">
+      <button class="button button__input">Body:</button>
+      <textarea class="body__input"></textarea>
+    </div>
+
+    <div class="positionframe">
+
+      <button class="button button__input">Position:</button>
+      <select class="position__input">
+        <option>Top Left</option>
+        <option>Top Right</option>
+        <option>Bottom Left</option>
+        <option>Bottom Right</option>
+      </select>
+
+      <div class="storypointframe">
+        <button class="button__storypoint">Storypoint<h2 class="storypoint__number">1</h2></button>
+        <button class="button__storypoint">Storypoint<h2 class="storypoint__number">2</h2></button>
+        <button class="button__storypoint--add">Add Storypoint<h2 class="storypoint__number storypoint__number--add">+</h2></button>
+      </div>
+
+    </div>
+
+    <br><br>
+
+      <div class="dimensions__field">
+
+        <h2>Fields</h2>
+
+        <div class="dragable">Years</div>
+        <div class="dragable">Years</div>
+        <div class="dragable">Foo</div>
+        <div class="dragable">Foo</div>
+
+
+      </div>
+
+      <br>
+
+      <div class="dimensions__variables">
```

Comment:
Same thing here with multiple components. The boxes where you drop stuff can be considered cards and should then use the `.card` class. The area where you actually drop things is a separate class and is essentially the same as the `.draggable--placeholder` above.